### PR TITLE
Keras 3 update - VAE

### PR DESCRIPTION
**Pull request by takumiohym**
Update summary:
- Updated to Keras 3
- Moved the KL divergence loss to the custom `Sampling` layer due to the [removal of symbolic `add_loss()`](https://keras.io/guides/migrating_to_keras_3/#removed-features).

### CODE REVIEW

**Review by sanjanalreddy — APPROVED** [review]

**Comment by takumiohym** [comment]
<p>Cleared. Thanks!</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/631/discussion/#comment-3165673526'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>
